Gemfile

```diff
@@ -36,6 +36,7 @@ gem 'jbuilder', '~> 1.2'
 gem 'devise', '>= 3.2.4'
 gem 'bootstrap-sass', '~> 3.3.0'
 gem 'autoprefixer-rails'
+gem 'rubocop', require: false
```

Comment:
maybe this should be in the development group block where the quiet_assets gem is since i don't think we need it in test or production